load_confounds.py

```diff
@@ -188,6 +230,8 @@ def load_confounds(
     pca_motion=1,
     wm_csf="basic",
     global_signal="basic",
+    compcor="anat",
+    n_compcor=-1,
```

Comment:
Two comments: the value 0 should translate to use all components, rather than -1. It would be consistent with `pca_motion`.

Also I suggest putting the default at 6, following this paper: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2214855/
Using all compcorr components is a catastrophy and should basically never been done, because you loose all your available degrees of freedom. 
The original method (and what I implemented in niak) was the "broken stick" test of significance on explained variance. But we would need to access the json files to expose this option to users, and at least for now I would rather avoid going that route.

Reply:
for posterity: if the user requests more components than available then whatever available components are provided and a warning is displayed.

Reply:
Using 0 for "all" is difficult because there is a 0 component. I removed the capability to use all components. Lmk what you think.